ci/deno.yml

```diff
@@ -16,22 +16,17 @@ on:
 
 jobs:
   test:
-    runs-on: ${{ matrix.os }} # runs a test on Ubuntu, Windows and macOS
-
-    strategy:
-      matrix:
-        deno: ["v1.x", "canary"]
```

Comment:
Should we keep the deno version matrix and only remove the OS matrix?

Reply:
This is only a quick start too, people can adapt their own pipelines based on their needs after getting kick started with this. I agree its nice to have the simplest, fasted and most stable setup for the initial pipeline template.